File: exercises/beecrowd/1154/1154.ts
// https://judge.beecrowd.com/pt/problems/view/1154
import * as fs from "fs";

function main(): void {
  const inputs = fs
    .readFileSync("/dev/stdin", "utf8")
    .trim()
    .split("\n")
    .map(Number);

  let sum = 0;
  let count = 0;

  for (const age of inputs) {
    if (age < 0) break;
    sum += age;
    count++;
  }

  const average = sum / count;
  console.log(average.toFixed(2));
}

main();

File: exercises/beecrowd/1154/1154.go
// https://judge.beecrowd.com/pt/problems/view/1154
package main

import "fmt"

func main() {
	var age, sum, count int

	for {
		fmt.Scan(&age)
		if age < 0 {
			break
		}
		sum += age
		count++
	}

	average := float64(sum) / float64(count)
	fmt.Printf("%.2f\n", average)
}
